### ISSUE TITLE
Release Twisted words 0.4.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11214]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1733|
|Type|task|
|Created|2006-05-21 23:27:12Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1733 1733
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1148254032000000 1148254032000000
changetime__1148257478000000 1148257478000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

+1 merge it

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16910]) News and Versioning for Twisted Words 0.4.0 release

Author: Release Automation, exarkun
Reviewer: exarkun, radix
Fixes [#11214](https://github.com/twisted/twisted/issues/11214) and [#11196](https://github.com/twisted/twisted/issues/11196)

This also removes the twisted.web.resource import from twisted.words.service, since it is
unused and creates a pointless dependency on twisted.web.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
